packages/py-gridtools-cpp/package.py

```diff
@@ -11,13 +11,17 @@ class PyGridtoolsCpp(PythonPackage):
 
     homepage = "https://gridtools.github.io/gridtools/latest/index.html"
 
-    pypi = "gridtools-cpp/gridtools-cpp-2.2.2.tar.gz"
+    pypi = "gridtools-cpp/gridtools-cpp-2.2.3.tar.gz"
 
     maintainers = ['samkellerhals']
 
     version("2.2.2",
```

Comment:
Do we need to keep this version?

Reply:
do you mean v2.2.2? Actually, I don't know. Probably just the latest version is required for the first release